pkg/mads/reconcile/snapshot_generator_test.go

```diff
@@ -210,13 +210,13 @@ var _ = Describe("snapshotGenerator", func() {
 						},
 						Spec: mesh_proto.Dataplane{
 							Networking: &mesh_proto.Dataplane_Networking{
-								Inbound: []*mesh_proto.Dataplane_Networking_Inbound{{
-									Interface: "192.168.0.3:3000:3000",
+								Address: "192.168.0.3",
```

Comment:
Is it expected that we change input data but the outcome doesn't change ? Why to change input data then ?

Reply:
This is a test for Gateway Dataplane. I wanted to test if it works after the change.